File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.kt
class Solution {
    fun solution(s: String): String {
        // 공백으로 구분된 숫자들을 리스트로 변환 후 정수로 변환
        val numbers = s.split(" ").map { it.toInt() }
    
        // 최소값과 최대값 찾기
        val min = numbers.minOrNull() ?: 0
        val max = numbers.maxOrNull() ?: 0
    
        // "(최소값) (최대값)" 형식의 문자열 반환
        return "$min $max"
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    # 공백으로 구분된 문자열을 정수 리스트로 변환
    numbers = list(map(int, s.split()))
    
    # 최소값과 최대값 찾기
    min_val = min(numbers)
    max_val = max(numbers)
    
    # "(최소값) (최대값)" 형식의 문자열 반환
    return f"{min_val} {max_val}"